### ISSUE TITLE
Is it possible to integrate Meeting Stones and the Level Range you need to be that you can use them ? Would be awesome, thanks!

### ISSUE THREAD

**Issue by baaatzi** [leatrix/leatrix-maps-classic#15]
Is it possible to integrate Meeting Stones and the Level Range you need to be that you can use them ? Would be awesome, thanks!

**Comment by leatrix** [comment]
Is this for Season of Mastery?  The dungeon icon tooltip (on the world map) should show the meeting stone level range (as shown on the meeting stone tooltip itself).  Have you found that to be incorrect?

**Comment by baaatzi** [comment]
Yes Season of Mastery, it is no longer displayed on the world map.

**Comment by leatrix** [comment]
Can you see the dungeon icons on the world map?  If you hover the pointer over the dungeon icon, you will see the level range for that dungeon in brackets.  Do you see ti?  For example, Stockade shows 24-32 in brackets.

**Comment by baaatzi** [comment]
No it only shows for example "Scarlet Monastery (34-45)" -> Next Line "Dungeon". Nothing more.

**Comment by leatrix** [comment]
But that's the level range in brackets which should match the meeting stone level range (as shown in the meeting stone tooltip, though I haven't verified them).  Is this not correct?

**Comment by baaatzi** [comment]
It is not correct. In the past it was shown corretly in two lines. First Line dunegon level requirement. Second line meeting stone level requirement. Think it is no longer working since Season of Mastery. An Update would be awesome. Thanks a lot.

**Comment by leatrix** [comment]
In BCC, it shows the minimum level required to enter the dungeon and to use the summoning stone.  This wasn't added to Classic Era because the meeting stones aren't summoning stones in Classic Era.  They are in SoM but SoM uses the Classic Era base.  If I see an accurate list of required levels (for the dungeon and meeting stones) for SoM, I can add them.  However, as the meeting stones didn't have this functionality in Classic Era, the listings around on the web seem to be for BCC only.
